src/client/package.json

```diff
@@ -44,7 +44,7 @@
   },
   "license": "MIT",
   "lint-staged": {
-    "src/**/*.{ts,tsx,json,scss}": [
+    "src/**/*.{js,ts,tsx,json,scss}": [
```

Comment:
We do not want to do this. No JS on this project, please.

Reply:
Because all code should be formatted — no matter what it is. Formatting code isn't an endorsement it's a comprehensive policy.